src/app/core/data/object-updates/object-updates.service.ts

```diff
@@ -330,4 +405,19 @@ export class ObjectUpdatesService {
   getLastModified(url: string): Observable<Date> {
     return this.getObjectEntry(url).pipe(map((entry: ObjectUpdatesEntry) => entry.lastModified));
   }
+
+  /**
+   * Get move operations based on the custom order
+   * @param url The page's url
+   */
+  getMoveOperations(url: string): Observable<MoveOperation[]> {
```

Comment:
Again, I think this one is missing from the specs.  The Spec changes for this service were a lot smaller than the service changes.